### ISSUE TITLE
🐛 Bug Report: Size validation not honoured for String attributes when an array

### ISSUE THREAD

**Issue by superseby2** [appwrite/appwrite#2703]
### 👟 Reproduction steps

- Create a collection "labels"
- Create an String "label" attribute with the following properties 
  - is_array -> true
  - size = 2
- Insert a document in collection "labels" using : {"label": ["ONE","2"] }

### 👍 Expected behavior

 - An exception because the validation should fail : label[0] has a size > 2

### 👎 Actual Behavior

- the document is inserted

### 🎲 Appwrite version

Version 0.12.x

### 💻 Operating system

Linux

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by aymericfontaine** [comment]
Same issue :

```
appwrite database getAttribute --collectionId=62120bd74259a75745f2 --key=testList
key : testList
type : string
status : available
required : false
array : true
size : 5
default :
```
Inserting a value longer than 5 characters should fail

**Comment by Meldiron** [comment]
Addressed in: https://github.com/utopia-php/database/pull/112

**Comment by stnguyen90** [comment]
This is a duplicate of https://github.com/appwrite/appwrite/issues/2681 and should be fixed now.

**Comment by Meldiron** [comment]
Fix released in 0.14 🥳 Don't hesitate to reopen the issue if you still encounter the same problem after upgrading.

**Comment by superseby2** [comment]
Works well ! Thx guys. 

On a side note : it might be interesting to be able to restrict the length of the array itself as well
